src/lib/Components/Bidding/Elements/Theme.tsx

```diff
@@ -0,0 +1,29 @@
+const Theme = {
+  // https://www.notion.so/artsy/Color-a0c24896daf8433d9409aee2146ac267
+  colors: {
+    text: "#000",
+    black100: "#000",
+    black80: "#333",
+    black60: "#666",
+    black30: "#C2C2C2",
+    black10: "#E5E5E5",
+    black5: "#F8F8F8",
+    purple100: "#6E1EFF",
+    purple30: "#D3BBFF",
+    green100: "#0EDA83",
+    yellow100: "#F1AF1B",
+    yellow30: "#FAE7BA",
+    yellow10: "#FDF7E8",
+    white100: "#FFF",
+  },
+
+  // https://www.notion.so/artsy/Spacing-93eeaed9fdf9480099fec7094fd1b9f3
+  space: [0, 3, 5, 10, 20, 30, 40, 60, 90, 120, 180],
```

Comment:
Yes but you can also refer to them by their pixel value.

From the docs:
```
// sets margin value of `theme.space[2]`
<Box m={2} />

// sets margin value of `-1 * theme.space[2]`
<Box m={-2} />

// sets a margin value of `16px` since it's greater than `theme.space.length`
<Box m={16} />
```

We'll be able to get IDE help for those values thanks to typescript

Reply:
In your example I think `m={4}` gets parsed by styled-system as "the array element at index 4 (i.e. the fifth element) of the `spaces` array" — so actually 20px 😐 

This will definitely take some getting used to.

Reply:
Oops, yeah. Ok, hmm. Would it be possible to move to a named specifier instead of an index?

Reply:
@l2success Thanks for clarifying. This does make me want a way to refer to these values by name, rather than index. What if we add new values later on? It’s possible I’m just not “getting” this approach yet, too.

Reply:
Since base units across services will remain the same, a common (but not necessary) approach would be to alias the values similar to how @yuki24 has done with the fonts. Of course, this could lead to some drift, but its easy enough to find / replace names and either way the base values underneath will remain the same no matter what the name is. Index's aren't required at the app level; they're just a convenient way to access primitives.

Reply:
Yeah that stuff about the (numerical) argument to the styled function being interpreted differently "since it's greater than `theme.space.length`" gives me pause. I fear this could be somewhat magical and error-prone. (Less so if these primitives never change, but not sure how realistic that is.)

I quickly tried using named (string) specifiers but [couldn't quite get that to work](https://codesandbox.io/s/387zmq7r4p).

The Typescript alternative that @acjay proposes also sounds interesting.

Reply:
Yah would ideally just want to stick with formal indexes